From dec03204800dbae3955477e498dfdcade11bb24b Mon Sep 17 00:00:00 2001
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Wed, 19 Nov 2014 11:22:53 +0000
Subject: [PATCH] Fix typo in the name of the info entry containing uptime

I'm assuming this is a typo, although it's been this way since it was added in r280

Signed-off-by: Jon Turney <jon.turney@dronecode.org.uk>
---
 src/client/windows/crash_generation/client_info.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/client/windows/crash_generation/client_info.cc b/src/client/windows/crash_generation/client_info.cc
index 75eca794..bb3e38ca 100644
--- a/src/client/windows/crash_generation/client_info.cc
+++ b/src/client/windows/crash_generation/client_info.cc
@@ -33,7 +33,7 @@
 #include "client/windows/crash_generation/client_info.h"
 #include "client/windows/common/ipc_protocol.h"
 
-static const wchar_t kCustomInfoProcessUptimeName[] = L"ptime";
+static const wchar_t kCustomInfoProcessUptimeName[] = L"uptime";
 static const size_t kMaxCustomInfoEntries = 4096;
 
 namespace google_breakpad {
-- 
2.53.0.windows.1

